getting_started/introduction/learning_new_features.rst

```diff
@@ -21,7 +21,7 @@ explore broad topics while the search bar will help you find more specific
 pages. If a page exists for a given theme, it will often link to more related
 content.
 
-.. image:: img/manual_search.png
+.. image:: img/manual_search.webp
```

Comment:
please don't change images to webp just for the sake of making them a webp file, we don't do that.

Reply:
Reverted.

Reply:
Reverted.

Reply:
We do encourage updating the format if the file itself is modified, as per our migration to webp

Reply:
Also reverted the image as well.

Reply:
I didn't upload this image, so as I understand it, the image should not be converted in this pull request.